crates/rattler_menuinst/data/menuinst.default.json

```diff
@@ -0,0 +1,68 @@
+{
+  "id_": "https://schemas.conda.io/menuinst-1.schema.json",
```

Comment:
This link seems to be broken

Reply:
Also, why is this file needed? :)

Reply:
We test that we can parse the defaults.json from upstream menuinst: https://github.com/conda/menuinst/blob/main/menuinst/data/menuinst-1-1-0.default.json

Reply:
I can't seem to find that test. Can you point me to where it is? :)

Reply:
```
    #[test]
    fn test_deserialize_defaults() {
        let test_data = test_data();
        let schema_path = test_data.join("defaults/defaults.json");
        let schema_str = std::fs::read_to_string(schema_path).unwrap();
        let schema: super::MenuInstSchema = serde_json::from_str(&schema_str).unwrap();
        insta::assert_debug_snapshot!(schema);
    }

```

Reply:
That uses `test-data/defaults/defaults.json`, not `data/menuinst.default.json`